Setting up affiliated pages and zfit (#11)
scikit-hep/scikit-hep.github.io: pages/affiliated.md

---
layout: page

> title: Affiliated Packages

Reviewer: ```suggestion
title: Affiliated packages
```

permalink: /affiliated
nav_order: 6
---

> Affiliated Packages {#affiliated_packages}

Reviewer: ```suggestion
Affiliated packages {#affiliated_packages}
```

====================

> In the following, projects that work closely together with scikit-hep are described. They extend the Python ecosystem in HEP and remain, due to their size and scope, generally independent of scikit-hep.

Reviewer: You should use "Scikit-HEP" for the project (2 occurrences in these 2 lines).

Author: done

zfit: scalable pythonic fitting
-------------------------------

[![img-zfit]][zfit]

The [zfit](https://zfit.github.io/zfit/) project brings together the different efforts of fitting in Python by providing a stable model

> fitting API and workflow. This comes together with an implementation of the fitting library that is based on TensorFlow, a deep learning framework built for high performance computing on modern architectures. While traditional functionalities such as loading ROOT-files or using the Minuit minimizer are integrated (thanks to scikit-hep packages), it is completely independent of the ROOT project and has no dependency on it.

Reviewer: Avoid "... together" again. How about "The implementation ..." instead of "This comes together with an implementation ..."?

Author: yes, true

> fitting API and workflow. This comes together with an implementation of the fitting library that is based on TensorFlow, a deep learning framework built for high performance computing on modern architectures. While traditional functionalities such as loading ROOT-files or using the Minuit minimizer are integrated (thanks to scikit-hep packages), it is completely independent of the ROOT project and has no dependency on it.

Reviewer: Add links for TensorFlow, ROOT and Minuit.

scikit-hep -> Scikit-HEP

Author: done

The core of zfit is a class-based approach for model building, composition and customization capabilities.

> The project consists of additional libraries which extend the capabilities of the core.

Reviewer: ```suggestion
The project consists of additional libraries extending the capabilities of the core package.
```
